packages/babel-plugin-check-constants/README.md

```diff
@@ -1,58 +0,0 @@
-# @babel/plugin-check-constants
-
-> Validate ES2015 constants (prevents reassignment of const variables).
```

Comment:
next pr: we should maybe include parts of this readme into the blocking scoping one?

Reply:
Good point, I can do this later tonight (today for you 😁)

Reply:
@hzoo I just added a note for the constants check in the readme #7279. Improvements to wording welcome! Anything else that is needed from a documentation perspective?